packages/playwright/src/generators/init/utils/update-targets.ts

```diff
@@ -39,21 +44,19 @@ export function updateProjectJsonWithNativeVisualRegressionTargets(
                     options: {
                         commands: [
                             {
-                                command:
-                                    'docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:v1.30.0-jammy npx nx run next-custom-server-e2e:e2e',
+                                command: `docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:${playwrightVersion}-jammy npx nx run ${project.name}:e2e`,
```

Comment:
I'm pretty sure you need to do something with the custom args here:

`docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:${playwrightVersion}-jammy npx nx run ${project.name}:e2e {args.extra}`

Reply:
Or is `--extra` a magic arg?

Reply:
--extra allows us to pass things like --update-snapshots for an "updatesnapshots" configuration to the mands executor it's only needed for that so far